docs/source/code/terrain_plot_aspect.py

```diff
@@ -0,0 +1,18 @@
+import matplotlib.pyplot as plt
+import xdem
+
+xdem.examples.download_longyearbyen_examples()
+
+dem = xdem.DEM(xdem.examples.FILEPATHS["longyearbyen_ref_dem"])
+
+aspect = xdem.terrain.aspect(dem.data)
+
+plt.imshow(aspect.squeeze(), cmap="jet")
```

Comment:
Just looking at the documentation, I thought jet is not the best colormap for aspect. It would be preferable to use a cyclic colormap, but I couldn't find a good one in matplotlib's defaults. If there is none, maybe we could make our own??
Then, I thought Raster.show() could be overwritten for the DEM class to have options to directly plot the hillshade, slope or aspect (with default colormap)! :-)

Reply:
I tried "hsv" first but that was absolutely terrible. "twiligt" is pretty nice in my opinion! What do you think?

Also, good idea with the plot override! Could also arguably fit in GeoUtils somehow.

![image](https://user-images.githubusercontent.com/33550973/123219624-63a3de80-d4cd-11eb-9579-23100f399c2e.png)